### PR TITLE
Improve highlight for selected conversation

### LINKED ISSUE #138: Better highlighting for current tab
First, great job!
Second, please make a better highlighting system for the current tab (current open user in the left pane).
I ended up sending some messages to a different user, since I couldn't quite visually distinguish between the currently selected and its neighboring tabs.

image

Cheers!

### PR DESCRIPTION
**Pull request by vinnyang**
Fixes #138 

- Used the blue from message bubbles with different opacity in
dark/light modes.
- Slight tweaked the color of the latest message subtitle, so it would be easier to read in light mode.

### CODE REVIEW

File: browser.css [hunk 1]
```diff
@@ -121,3 +121,13 @@ a,
...
+
+/* Contact list: person (selected) */
+._1ht1._1ht2 {
+	background-color: rgba(0, 132, 255, 0.8) !important;
+
+}
+/* Contact list: selected person, latest message */
+._1ht1._1ht2 ._1htf {
+	color: rgba(0, 0, 0, 0.4);
+}
```
Comment (sindresorhus) [inline browser.css]: This doesn't actually make it any more readable in either light or dark mode.

File: browser.css [hunk 2]
```diff
@@ -121,3 +121,13 @@ a,
 ._4eby {
 	border-radius: 2px !important;
 }
+
+/* Contact list: person (selected) */
+._1ht1._1ht2 {
+	background-color: rgba(0, 132, 255, 0.8) !important;
```
Comment (sindresorhus) [inline browser.css]: `0.8` is too strong of a color. I would go with `0.3`.

File: dark-mode.css [hunk 3]
```diff
@@ -138,7 +138,7 @@ html.dark-mode ._1ht1 {
 
 /* Contact list: person (selected) */
 html.dark-mode ._1ht1._1ht2 {
-	background: rgba(255, 255, 255, 0.03) !important;
+	background-color: rgba(0, 132, 255, 0.6) !important;
```
Comment (sindresorhus) [inline dark-mode.css]: I think you instead should match the macOS dark mode highlight color, which is a gradient that starts at #1871DC and ends at #1870DB.

File: browser.css (hunk 1 above)
Comment (vinnyang) [inline browser.css]: @sindresorhus what color would you suggest for this?

File: dark-mode.css (hunk 3 above)
Comment (vinnyang) [inline dark-mode.css]: Sure, I will change that.

File: browser.css (hunk 1 above)
Comment (vinnyang) [inline browser.css]: @sindresorhus I reverted this change, that latest message bit seems pretty readable in both modes with the new background colors you suggested.

**Comment by sindresorhus** [comment]
Looks pretty good now, but I think the https://github.com/sindresorhus/caprine/pull/179#discussion_r111705760 text needs to be a bit darker in light mode, as it's not very legible.

In dark mode, the date screen shot 2017-04-17 at 16 04 53 when selected
 needs to be a bit lighter.

**Comment by vinnyang** [comment]
@sindresorhus updated accordingly, please check again when you have a chance, thanks!

File: dark-mode.css [hunk 4]
```diff
@@ -138,7 +138,7 @@ html.dark-mode ._1ht1 {
 
 /* Contact list: person (selected) */
 html.dark-mode ._1ht1._1ht2 {
-	background: rgba(255, 255, 255, 0.03) !important;
+	background: linear-gradient(-45deg, #1871DC, #1870DB) !important
```
Comment (sindresorhus) [inline dark-mode.css]: Should be `linear-gradient(180deg, #1871DC, #1870DB) !important;`

**Comment by sindresorhus** [comment]
Looks great. Thank you! :)

**Pull request merged**